src/webui/www/private/scripts/mocha-init.js

```diff
@@ -810,6 +813,92 @@ const initializeWindows = function() {
         }
     };
 
+    resumeTorrentsByTrackerFN = function(trackerHash) {
+        const trackerHashInt = Number.parseInt(trackerHash, 10);
+        let hashes = [];
+        switch (trackerHashInt) {
+            case TRACKERS_ALL:
+                hashes = torrentsTable.getFilteredTorrentsHashes('all', CATEGORIES_ALL, TAGS_ALL, TRACKERS_ALL);
+                break;
+            case TRACKERS_TRACKERLESS:
+                hashes = torrentsTable.getFilteredTorrentsHashes('all', CATEGORIES_ALL, TAGS_ALL, TRACKERS_TRACKERLESS);
+                break;
+            default:
+                hashes = trackerList.get(trackerHashInt).torrents
```

Comment:
Let's add a `break;` to every `default:` cases.
Don't forget the new code in dynamicTable.js.